app/controllers/deliveries/show/resend.js

```diff
@@ -0,0 +1,37 @@
+import Ember from 'ember';
+import CanResendController from './can-resend-controller';
+
+export default CanResendController.extend({
+  errors: [],
+  errorMessages: Ember.computed('errors', function () {
+    const errorMessages = [];
+    this.get('errors').forEach(function (generalError) {
+      errorMessages.push(generalError.detail);
+    });
+    return errorMessages;
+  }),
+  actions: {
+    resend() {
+      const thisController = this;
+      function consumeError(error) {
+        thisController.set('errors', error.errors);
+      }
+      const transfer = this.get('model');
+      const projectName = transfer.get('project.name');
+      const deliveryMessage = 'Email message resent for delivery of project ' + projectName + '.';
+      // save user message changes
+      transfer.get('delivery').then(function (delivery) {
```

Comment:
On my local dev instance, I don't have a `delivery` for every `transfer` provided by DukeDS. In that scenario, this line logs an error `null is not an object (evaluating 'delivery.save')` but no error is shown in the UI.

I recognize this is not a common scenario, but it's certainly possible, and should be easy enough to trap that error as well.